crates/core/src/subscription/execution_unit.rs

```diff
@@ -71,6 +71,7 @@ enum EvalIncrPlan {
 pub struct ExecutionUnit {
     hash: QueryHash,
 
+    pub(crate) sql: String,
```

Comment:
Does this need to be public?

Reply:
This one yes, is read elsewhere and making a helper method not buy much.